spec/class.dd

```diff
@@ -633,10 +633,10 @@ $(GNAME Destructor):
         )
 
         $(P The  program can explicitly inform the garbage collector that an
-        object is no longer referred to (with the delete expression), and
+        object is no longer referred to
+        (with $(LINK2 phobos/object.html#.destroy, $(D object.destroy()))), and
```

Comment:
`$(REF1 destroy, object)
or if you like the parenthesis sth. like $(REF1_ALTTEXT, $(D object.destroy()), destroy, object)